### PR TITLE
TEJ-1136: guava -> 18.0; icu -> 59.1

### PR DESCRIPTION
**Pull request by Katsuya-Tomioka**

### CODE REVIEW

**Comment by Katsuya-Tomioka** [comment]
all green. now running travis openjdk7/8 and oraclejdk8.

File: pom.xml [hunk 1]
```diff
@@ -77,7 +77,7 @@
         <dependency>
             <groupId>com.ibm.icu</groupId>
             <artifactId>icu4j</artifactId>
-            <version>53.1</version>
+            <version>59.1</version>
```
Comment (iredpath) [inline pom.xml]: open-source-parent manages this, but is out of sync with the internal parent.  Think its worth bumping and releasing internally?

File: pom.xml [hunk 2]
```diff
@@ -87,7 +87,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>16.0.1</version>
+            <version>18.0</version>
```
Comment (iredpath) [inline pom.xml]: like above, this is managed (though this time correctly) in open-source-parent; you can just bump that to 1.1.8 and remove this version declaration

Comment (Katsuya-Tomioka) [reply pom.xml]: 👍

**Comment by Katsuya-Tomioka** [comment]
should be all set.

File: pom.xml [hunk 3]
```diff
@@ -48,12 +48,10 @@
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-core</artifactId>
             <version>1.3</version>
-            <scope>test</scope>
```
Comment (iredpath) [inline pom.xml]: @Katsuya-Tomioka did you mean to remove the `version` tags here?

**Pull request merged**